github/index.ts

```diff
@@ -121,35 +121,95 @@ export class GitHubClient {
     })
   }
 
+  private issueParts(issueUrl) {
```

Comment:
in some scenarios I have the apiUrl and in some (extracting url from body) I have the html_url so centralized